Dockerfile

```diff
@@ -58,15 +59,18 @@ RUN cd /sandbox && for package_name in \
 		collections \
 		automem \
 		pegged \
+		dpp \
 		; do \
       	package="$(echo $package_name | cut -d: -f1)"; \
       	version="$(echo $package_name | grep : |cut -d: -f2)"; \
       	version="${version:-*}"; \
 		printf "/++dub.sdl: name\"foo\"\ndependency\"${package}\" version=\"${version}\"+/\n void main() {}" > foo.d; \
 		dub fetch "${package}" --version="${version}"; \
 		dub build --single -v --compiler=${DLANG_EXEC} foo.d; \
-		version=$(dub describe ${package} | jq '.packages[0].version') ; \
-		echo "${package}:${version}" >> packages; \
+		if [ "${package}" != "dpp" ] ; then \
+			version=$(dub describe ${package} | jq '.packages[0].version') ; \
+			echo "${package}:${version}" >> packages; \
+		fi ; \
```

Comment:
The package list is parsed by run.dlang.io for the "Add package" feature.
It might be easier to issue `dub fetch dpp && dub build --compiler=${DLANG_EXEC} dpp` after the loop.